source/host.c

```diff
@@ -547,10 +557,14 @@ Returns false if the time is too short to run a frame
 qboolean Host_FilterTime (float time)
 {
 	realtime += time;
-
+#ifndef __WII__
    if (cl_maxfps.value < 1) Cvar_SetValue("cl_maxfps", 30);
    if (!cls.timedemo && realtime - oldrealtime < 1.0/cl_maxfps.value)
 		return false;		// framerate is too high
+#else
+	if (!cls.timedemo && realtime - oldrealtime < 1.0f/72.0f)
```

Comment:
`72.0f` should be the current framerate. So `30` or `60` depending on your target.